ReactSkia/BUILD.gn

```diff
@@ -116,6 +116,9 @@ source_set("ReactSkia") {
     "core_modules/RSkDeviceInfo.h",
     "core_modules/RSkEventEmitter.cpp",
     "core_modules/RSkEventEmitter.h",
+    "core_modules/RSkKeyboardObserver.h",
```

Comment:
should these files be included under "rns_enable_onscreen_keyboard" like OnScreenKeyBoard.cpp file